File: unit2/selectionsort.c
#include<stdio.h>
#include<stdlib.h>
int main()
{
	int n;
	scanf("%d",&n);
	int* arr=calloc(sizeof(int),n);
	for(int i=0;i<n;i++)
	{
		scanf("%d",&arr[i]);
	}
	int minpos=0;
	for(int i=0;i<n-1;i++)
	{
		minpos=i;
		for(int j=i+1;j<n;j++)
		{
			if(arr[minpos]>arr[j])
				minpos=j;
		}
		int t=arr[minpos];
		arr[minpos]=arr[i];
		arr[i]=arr[minpos];
	}
	for(int i=0;i<n;i++)
	{
		printf("%d ",arr[i]);
	}
	return 0;
}

File: unit2/selectionsort.py
s=input().split()
arr=[(x,y) for x,y in enumerate (list(map(int,s)))]
print(arr)
for i in range(len(arr)-1):
    minpos=i
    for j in range(i+1,len(arr)):
        if(arr[minpos][1]>arr[j][1]):
            minpos=j
    arr[minpos],arr[i]=arr[i],arr[minpos]
print(arr)

